front/src/pages/App/App.tsx

```diff
@@ -2,11 +2,12 @@ import React from 'react';
 import { Routes, Route } from 'react-router-dom';
 import Header from './Header/Header';
 import Footer from './Footer';
-import Home from '../Home';
+import Home from '../Home/Home';
```

Comment:
we can make `Home/index.tsx` which would just re-export Home from `Home.tsx`

Then, I believe, we will be able to import component as `import Home from '../Home'` - looks like "importing from folder" but actually "importing from index file in the folder". Why not putting component into "index.tsx" then you may ask me? It's just more convenient to search file by "Home" in IDE, than typing "index.tsx" and choose between dozens of them, in different folders.

Not proposing to refactor all the code, but propose to consider such an approach